docs/source/conf.py

```diff
@@ -182,7 +186,7 @@
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [(master_doc, "pyansys", "PyAnsys DPF-Post Documentation", [author], 1)]
+man_pages = [(master_doc, "PyDPF-Post", "PyAnsys DPF-Post Documentation", [author], 1)]
```

Comment:
what does that mean?

Reply:
@cbellot000 It is the name of the Tex file created, and thus of the pdf file, giving us a "PyDPF-Post.pdf" file for the manual.